portfolio/src/main/webapp/script.js

```diff
@@ -85,8 +85,13 @@ function goBack() {
 /**
  * Fetches the response of "/data".
  */
-function loadAndShowData() {
-    fetch("/comment").then(response => response.json()).then((json) => {
+function loadAndShowComments() {
+    let maxNumberOfComments = document.getElementById("quantity").value;
+    if (maxNumberOfComments.length == 0) {
+        maxNumberOfComments = 0;
```

Comment:
Does it make sense to fetch with max return 0?

Reply:
i design this so that if the user does not enter anything but press the submit button, it will change the maxNumberOfComments to 0 and if there are existing shown comments, they will disappear